frontend/public/components/utils/units.js

```diff
@@ -147,6 +147,78 @@ units.dehumanize = (value, typeName) => {
   return convertValueWithUnitsToBaseValue(value, type.units, type.divisor);
 };
 
+const humanizeUsageUnit = (unit, type, singular) => {
```

Comment:
Why can't we use the existing humanize methods? There is already `units.humanize`

Reply:
Actually, I'd suggest we should just show the string value as-is since that's what we do in other places in the console. Then we can use `units.dehumanize` for comparing values and calculating percent usage.

Reply:
> Actually, I'd suggest we should just show the string value as-is since that's what we do in other places in the console.

For example,

![okd 2018-11-16 14-02-38](https://user-images.githubusercontent.com/1167259/48641624-5e429700-e9a8-11e8-84dd-7eb7fba864b6.png)

We should show these the same way here to avoid confusion.

Reply:
@spadgett, ok with showing units 'as-is', but wondering about the CPU units of 'cores'.  Max CPU only shows an integer (ex. '1') should this be converted to `1000 m'? -thanks

Reply:
I would leave it as-is for now at least.

Reply:
fixed